script.js

```diff
@@ -0,0 +1,27 @@
+//Start when page loads
+//Target elements in HTML to access in JS file
+//define variables to keep track of whose turn it is and who wins each round
+//define variabls for X and O moves
+
+window.onload = start;
+var boxes = document.getElementsByClassName("square");
+var moveTracker = 1;
+var winTracker = 0;
+var xMoves = [];
+var oMoves = [];
+
+//Set up event listener to listen for click
+
+for (var 1 = 0; i < boxes.length; i++) {
```

Comment:
Check your console! It throws this error: `Uncaught SyntaxError: Unexpected number` You have `var 1 = 0`. You can't use `1` as the name for a variable. Right now Javascript thinks you're trying to tell it that 1 is equal to 0, which is certainly isn't!